en/mozorg/home-new.ftl

```diff
@@ -23,20 +23,46 @@ home-product-firefox-browsing = Private & secure browsing
 home-cta-get-firefox = Get { -brand-name-firefox }
 home-product-pocket-articles = The web’s most intriguing articles
 home-cta-get-pocket = Get { -brand-name-pocket}
+
+# Obsolete string (expires: 2024-07-16)
 home-product-relay-masks = Easy-to-use email & phone masks
+
+home-product-relay-hide-your = Hide your phone number and email from spammers
 home-cta-get-relay = Get { -brand-name-relay }
 home-product-vpn-trust = A VPN you can trust
 home-cta-get-vpn = Get { -brand-name-mozilla-vpn }
+
+# Obsolete string (expires 2024-07-23)
 home-product-monitor-data = Data breach alerts
+
+home-product-monitor-protect-your = Protectr your private info from data brokers
 home-cta-get-monitor = Get { -brand-name-monitor }
+home-product-fakespot-detect = Detect fake shopping reviews
+home-cta-get-fakespot = Get { -brand-name-fakespot }
+
 
 home-mozilla-takes-bets = “{ -brand-name-mozilla } is taking bets to show the world there’s a business to be made with trustworthy AI. That includes putting things like human rights, data protection and transparency at the core of how these complex systems work.”
 
 # Politico is a brand name for a news organization
 home-politico-cite = Politico
 
+home-join-us-in-shaping = Join us in shaping trustworthy AI
+home-work-on-ai = { -brand-name-mozilla }’s work with AI isn’t just a new thing—we’ve spent years funding, building and advocating for AI that’s open, fair and developed responsibly. Our focus is on creating AI that serves the people, prioritizes transparency and supports the public good, not corporate agendas.
```

Comment:
This:
> "work with AI isn’t just a new thing—we’ve spent years funding"

already is `U+2014` EM DASH

@peiying2 Do you mean another occurence?

Reply:
The only inconsistent thing here is that while this sentence uses the dash correctly — without the spaces on both sides (at it should be IIRC), the other string on L64 has spaces around it. So the only thing I see here wrt dashes would be making these two occurrences more consistent: https://github.com/mozilla/bedrock/pull/14602#discussion_r1614789642

Reply:
OTOH the "correct" (non)spacing produces some ugly line breaks at times:

![Screenshot 2024-05-25 at 20 24 02](https://github.com/mozilla-l10n/www-l10n/assets/1784648/83a5ca65-39ac-416f-8bdc-dc7621564949)

so the question is whether to give up and just have spaces around it?

Reply:
I've asked the copy writing team for clarification on spaces and em dashes. It seems we are doing it a couple different ways currently and since we are trying to go a little more colloquial with our tone, that might be true of our grammar too so I don't feel confident making a call myself.

Reply:
Oh, it was hard to see it. I agree with @janbrasna that there should be a space before and after the M dash, but it has been inconsistent throughout. If there is a branding or style guide somewhere, let me know. For now, I will take this as is and move forward.

Reply:
I talked to someone in brand and they said spaces around the dash are the way to go.

They also said they don't have a style guide.